conf/app.routes

```diff
@@ -1,3 +1,5 @@
 # microservice specific routes
 
-POST        /claim         uk.gov.hmrc.cdsreimbursementclaim.controllers.SubmitClaimController.claim
\ No newline at end of file
+POST        /claim                   uk.gov.hmrc.cdsreimbursementclaim.controllers.SubmitClaimController.claim
+
+POST        /declaration-info       uk.gov.hmrc.cdsreimbursementclaim.controllers.DeclarationInfoController.getDeclarationInfo
```

Comment:
Rename to `POST /declaration`

Reply:
At the moment the frontend passes the whole json structure down.
To change it to just a simple declarationID, I will have to generate the whole json here.